src/android/CameraPreviewFragment.java

```diff
@@ -114,7 +119,26 @@ public void startCamera() {
         }
     }
 
+    public void torchSwitch(boolean on, TorchCallback torchCallback) {
```

Comment:
```suggestion
    public void torchSwitch(boolean state, TorchCallback torchCallback) {
```
un nom un peu plus général, je propose state comme on le fait pour iOS